internal/service/networkpeering/resource_network_peering_test.go

```diff
@@ -18,32 +18,45 @@ import (
 )
 
 func TestAccNetworkRSNetworkPeering_basicAWS(t *testing.T) {
```

Comment:
```suggestion
func TestAccNetworkNetworkPeering_basicAWS(t *testing.T) {
```
Just to make explicit that this test includes both resource and data sources